.github/workflows/ci.yml

```diff
@@ -78,6 +79,39 @@ jobs:
         run: yarn test-unit
       - name: Integration Tests
         run: yarn test-integration
+  test-ibmi:
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [12]
```

Comment:
The issue I was running into is that both actions were running against the same user/schema, so they were creating/deleting each other's tables (the pitfalls of testing against an external DB). I think that I have multiple users set up now, just need to adjust the action so that they use their own. Still could be an issue when they use an explicit schema.